### ISSUE TITLE
Dasiy ACE Javascript error

### ISSUE THREAD

**Issue by JulietOSU** [daisy/ace-gui#103]
Every time I try to use this product, I get this Javascript error:

![image](https://user-images.githubusercontent.com/61983680/76235395-69ef4800-6201-11ea-87ba-b6fcc3261557.png)
And then it finds no errors.
When I upload a different file, I get the same error and the same results. It will not overwrite the previous report unless I close & reopen the application. Same Javascript error.

So this effectively does not work.
Windows 10
![image](https://user-images.githubusercontent.com/61983680/76235562-a622a880-6201-11ea-8ee2-39213a4eb7d9.png)

Please provide the following details if possible or relevant:

- [] the version of Ace impacted by the issue (as returned by the `ace -v` command).
- [] the version of Node used to run Ace (as returned by the `node -v` command).
- [] your operating system and version.
- [] if the issue is a bug when running Ace: the error message as produced in the log file (you can enable debug logging with the `--verbose` option).
- [] if the issue is about the HTML report: the browser(s) and version(s) you used.
- [] if possible, let us know if you can share a sample input file to help us reproduce the issue.

**Comment by danielweck** [comment]
Hello, would you mind trying the latest build from this link please:
https://github.com/daisy/ace-gui/releases/tag/latest-windows
(the filename should be `Ace.by.DAISY.Setup.1.0.2-alpha.1.59.exe` or another future increment such as `-alpha.1.60`, `-alpha.1.61` etc.)

**Comment by JulietOSU** [comment]
Symantec is saying the file isn't safe and "growing evidence that the 
file is untrustworthy."

**Comment by danielweck** [comment]
Yes, standard procedure for Windows and Mac OS to report apps that are not signed with recognized certificates from reputable organizations. In this particular case, the latest automated builds of Ace App are not signed at all, so there are additional warnings when the app is installed and launched.

**Comment by JulietOSU** [comment]
.exe doesn't run.  It says I need an app to install it.  And no, I don't 
want to search for an app in the store.

**Comment by danielweck** [comment]
Hello, what the exact error message?
With unsigned apps in Windows, there is a warning message that appears in a popup dialog box which the user can expand in order to reveal an additional button that allows the application to install / run.

**Comment by DaveGunn** [comment]
I had the same JavaScript error with the Ace 1.1.0, triggered when viewing the Outlines tab, but only happened once and could not be replicated.
![Ace-javascript-error](https://user-images.githubusercontent.com/8447023/80304741-ea690880-87af-11ea-93cd-29c2916ddd68.jpg)
